global-script.lua

```diff
@@ -3890,3 +3896,34 @@ function onPlayerDisconnect(player)
         updatePlaySpiritButton(player.color)
     end
 end
+
+function onObjectSpawn(obj)
+    local is_power_card = obj.hasTag("Minor") or obj.hasTag("Major") or obj.hasTag("Unique")
+    if is_power_card then
```

Comment:
can you remove the local variable and just put the conditional here?

Reply:
I think it reads more clearly like this, but I don't feel strongly. Will change it as a batch with my other change, once we settle on it.

Reply:
Probably out of scope for this PR, but I think it would be clearer if there was a tag for being a power card (in addition to the minor/major/unique tags). That way, the condition would be `if obj.hasTag("Power")`.

Reply:
Is there such a thing as a tag hierarchy? It makes me a bit queasy for tags to imply other tags without there being a formal relationship.

Reply:
current there is no concept of a tag heirarchy, but there could just be a global function called `isPowerCard` that looks for those tags that could be reused throughout the code

Reply:
Removed.